modules/swagger-codegen/src/main/resources/python/api.mustache

```diff
@@ -1,22 +1,5 @@
 # coding: utf-8
 
-"""
-{{classname}}.py
-Copyright 2015 SmartBear Software
-
-   Licensed under the Apache License, Version 2.0 (the "License");
-   you may not use this file except in compliance with the License.
-   You may obtain a copy of the License at
-
-       http://www.apache.org/licenses/LICENSE-2.0
-
-   Unless required by applicable law or agreed to in writing, software
-   distributed under the License is distributed on an "AS IS" BASIS,
-   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-   See the License for the specific language governing permissions and
-   limitations under the License.
-"""
-
```

Comment:
Yea. The license is not present in the scala code. Either it should be added in those places or removed here.

In principle I think you want to NOT have a license in these files, because that seriously undermines the usefulness of the tool. A commercial company probably wouldn't want to ship an SDK with a copyright other than that company.